src/views/Teams/index.tsx

```diff
@@ -1,24 +1,26 @@
 import React from 'react'
 import { AutoRenewIcon, Flex, Heading } from '@pancakeswap/uikit'
 import orderBy from 'lodash/orderBy'
-import { useTeams } from 'state/teams/hooks'
+import useSWRImmutable from 'swr/immutable'
 import Page from 'components/Layout/Page'
 import { useTranslation } from 'contexts/Localization'
 import TeamListCard from './components/TeamListCard'
 import TeamHeader from './components/TeamHeader'
+import { getTeams } from '../../state/teams/helpers'
+import { FetchStatus } from '../../config/constants/types'
 
 const Teams = () => {
   const { t } = useTranslation()
-  const { teams, isLoading } = useTeams()
-  const teamList = Object.values(teams)
+  const { data, status } = useSWRImmutable('teams', async () => getTeams())
```

Comment:
looks like if we use `useSWRImmutable` here, it won't trigger fetch on client

Reply:
Done 👍 changed to useSwr